packages/web/components/Landing/IntroHero.tsx

```diff
@@ -0,0 +1,69 @@
+import { Button, Heading, Stack, Text } from '@metafam/ds';
+// import { animated } from '@react-spring/web';
+import { useOnScreen } from 'lib/hooks/useOnScreen';
+import { useRouter } from 'next/router';
+import { useRef } from 'react';
+
+export const IntroHero: React.FC = () => {
+  const { push } = useRouter();
+  const heroRef = useRef<HTMLDivElement>(null);
+  const onScreen = useOnScreen(heroRef);
+
+  const handleSectionNav = (sectionId: string) => {
```

Comment:
can we have more descriptive section ids? 
seeing `#section-1` or `#section-2` in the url isn't exactly great imho.